### PR TITLE
feat(distribute): expand overview page

### PR DESCRIPTION
**Pull request by lucasfernog**
- split links per platform
- added information on build vs bundle
- added versioning section

### CODE REVIEW

File: src/content/docs/distribute/index.mdx [hunk 1]
```diff
@@ -47,64 +105,126 @@ Signing is required on most platforms. See the documentation for each platform f
...
+    href="/distribute/snapcraft/"
+    description="Distribute on Snapcraft.io"
   />
+</CardGrid>
+
+<LinkButton href="/distribute/sign/linux/">Code signing</LinkButton>
+
+### macOS
+
+For macOS you can either distribute your application directly to the App Store or ship a DMG installer as direct downloads.
```
Comment (vasfvitor) [inline src/content/docs/distribute/index.mdx]:
```suggestion
For macOS you can either distribute your application directly to the App Store or ship a DMG installer as direct download.
```

File: src/content/docs/distribute/index.mdx [hunk 2]
```diff
@@ -3,12 +3,70 @@ title: Distribute
...
+
+## Versioning
+
+Your application version can be defined in the [`tauri.conf.json > version`] configuration option,
+which is the recommended way for managing the app version. If that config value is not set,
+Tauri uses the `package > version` value from your `src-tauri/Cargo.toml` file instead.
+
+:::note
+Some platforms have some limitations and special cases for the version string.
+See the individual distribution documentation pages for more information.
```
Comment (vasfvitor) [inline src/content/docs/distribute/index.mdx:68]:
```suggestion
See the individual distribution documentation pages for more information.
```
idk, this sentence sounds odd to me but I don't want to repeat "platform": 

"See the specific platform documentation pages for more information"

**Review by vasfvitor — APPROVED** [review]

File: src/content/docs/distribute/index.mdx (hunk 2 above)
Comment (lucasfernog) [reply src/content/docs/distribute/index.mdx:68]: looks good

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Pull request merged**